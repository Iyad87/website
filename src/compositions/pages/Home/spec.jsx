import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './index';

test('compositions/homepage/Home snapshot', () => {
  const wrapper = shallow(<HomePage />);

  expect(wrapper).toMatchSnapshot();
});
