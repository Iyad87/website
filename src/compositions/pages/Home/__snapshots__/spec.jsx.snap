// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/Home snapshot 1`] = `
<React.Fragment>
  <BuildBetter />
  <CaseStudies />
</React.Fragment>
`;
